### ISSUE TITLE
Do I understand correctly?

### ISSUE THREAD

**Issue by bearpku** [vijaykatam/django-cache-manager#27]
1. If I changed an object, the whole cached objects related to the same table will be invalidated.
2. when we invalidate table, we just change the namespace, we will keep the stale cached objects in the memory, but no one will get that stale objects.

**Comment by vijaykatam** [comment]
Yes. Any change to a model or a related model(foreign key, many to many) will invalidate entries by creating a new key. The stale entries remain in memory and will not be accessible via cache manager. Memcache or redis will flush them eventually as new entries get added and referenced.

**Comment by bearpku** [comment]
Why django-cache-manager didn't overwrite the create method in query set? just bulk_create?
